cmd/tailscale/main.go

```diff
@@ -434,6 +437,15 @@ func (a *App) runBackend() error {
 			case ExitAllowLANEvent:
 				state.Prefs.ExitNodeAllowLANAccess = bool(e)
 				go b.backend.SetPrefs(state.Prefs)
+			case UseTailscaleDNSEvent:
+				state.Prefs.CorpDNS = bool(e)
+				go b.backend.SetPrefs(state.Prefs)
+			case UseTailscaleSubnetsEvent:
+				state.Prefs.NoSNAT = !bool(e)
```

Comment:
This should be state.Prefs.RouteAll

NoSNAT is the preference for `tailscale up --snat-subnet-routes` on Linux.

Reply:
Thanks for the input. I will adapt this ASAP!